grails-app/domain/com/asinha/domain/Customer.groovy

```diff
@@ -27,16 +27,16 @@ class Customer extends BaseEntity{
     String state
 
     static constraints = {
-        address size: 1..255, nullable: false
-        addressNumber size: 1..255, nullable: false
-        city size: 1..255, nullable: false
-        complement size: 1..255, nullable: true
-        cpfCnpj size: 11..14, nullable: false
-        email email: true, size: 5..255, nullable: false
-        phone size: 1..255, nullable: false
-        name size: 1..255, nullable: false
-        postalCode size: 1..255, nullable: false
-        province size: 1..255, nullable: false
-        state size: 1..255, nullable: false
+        address nullable: false
```

Comment:
Quando não informamos nada, o padrão é nullable false, então a gente costuma preencher só no cenário inverso, quando precisa ser nullable true. Nesses casos aqui não precisaria colocar nada, por padrão ja iria ser criado como not null